config/locales/self_service.en.yml

```diff
@@ -7,7 +7,8 @@ en:
     save_button: Save
 
     advisers_index:
-      title: "%{firm_name} Advisers"
+      title: Advisers
+      title_full: "%{firm_name} Advisers"
```

Comment:
Seems like `title_full` is the canonical title here — since it's being used in the `h1`. Could we swap the nomenclature? Maybe `title` and `breadcrumb` is closer to the truth?